### PR TITLE
add val models

### PR DESCRIPTION
**Pull request by mauricefallon**
As referred to in https://github.com/RobotLocomotion/drake/pull/1468 , I'm going to start adding Valkyrie with balancing and walking examples. The approach is mirrored off of what exists for Atlas.

This first pull request is to add the urdf and the meshes. Thanks to @MarcoCar 's work, the model is only 7.1MB. (Atlas is 46MB, IRB 7.5MB)

The only files touched outside of the Valkyrie directory are:
* drake/util/rospack.m
* drake/examples/CMakeLists.txt

The simple script reads the urdf and displays the model in the visualizer:
 drake/examples/Valkyrie/runValkyrieVisualize.m

### CODE REVIEW

File: drake/examples/Valkyrie/Valkyrie.m [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+      if ~isfield(options,'hand_right')
+        options.hand_right = 'none';
+      end
+      if ~isfield(options,'hand_left')
+        options.hand_left = 'none';
+      end
+      if ~isfield(options,'external_force')
+        options.external_force = [];
+      end
+      if ~isfield(options,'atlas_version')
```
Comment (MarcoCar) [inline drake/examples/Valkyrie/Valkyrie.m]: If that's needed we should rename it to valkyrie_version

File: drake/examples/Valkyrie/Valkyrie.m [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+        hand_position_right = [0; -0.195; -0.01];
+        hand_position_left = hand_position_right;
+        hand_orientation_right = [0; -pi/2; pi];
+        hand_orientation_left = [0; pi/2; pi];
+      elseif options.atlas_version == 5
+        hand_position_right = [0; -0.195; 0.0];
+        hand_orientation_right = [0; -pi/2; pi];
+        hand_position_left = [0; -0.195; 0.0];
+        hand_orientation_left = [0; -pi/2; pi];
+      end
```
Comment (MarcoCar) [inline drake/examples/Valkyrie/Valkyrie.m]: maybe that's not needed for val

**Comment by mauricefallon** [comment]
Please ignore the cruft in Valkyrie.m (taken from Atlas.m), I will clean it up with a follow pull request
containing working balancing and walking tests.

**Comment by mauricefallon** [comment]
Any chance of a quick accept of this? It only contains the 3D models and I'd like to follow up with the walking and balancing example tests before January.

File: drake/examples/Valkyrie/Valkyrie.m [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+      if ~isfield(options,'dt')
+        options.dt = 0.001;
+      end
+      if ~isfield(options,'floating')
+        options.floating = true;
+      end
+      if ~isfield(options,'terrain')
+        options.terrain = RigidBodyFlatTerrain;
+      end
+      if ~isfield(options,'hand_right')
```
Comment (RussTedrake) [inline drake/examples/Valkyrie/Valkyrie.m]: the hands should not be optional here, either.

File: drake/examples/Valkyrie/Valkyrie.m [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+        options.atlas_version = 5;
+      end
+
+      w = warning('off','Drake:RigidBodyManipulator:UnsupportedVelocityLimits');
+
+      obj = obj@TimeSteppingRigidBodyManipulator(urdf,options.dt,options);
+      obj = obj@Biped('r_foot_sole', 'l_foot_sole');
+      warning(w);
+
+      if options.atlas_version == 3
```
Comment (RussTedrake) [inline drake/examples/Valkyrie/Valkyrie.m]: zap all this

File: drake/examples/Valkyrie/Valkyrie.m [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+    hand_left_kind = 'none';
+    % preconstructing these for efficiency
+    left_full_support
+    left_toe_support
+    right_full_support
+    right_toe_support
+    left_full_right_full_support
+    left_toe_right_full_support
+    left_full_right_toe_support
+    atlas_version = [];
```
Comment (RussTedrake) [inline drake/examples/Valkyrie/Valkyrie.m]: zap me

File: drake/examples/Valkyrie/Valkyrie.m [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+
+      weights = struct('relative', [1;1;1;0;0;0.5],...
+                       'relative_final', [10;10;10;0;0;2],...
+                       'goal', [100;100;0;0;0;10]);
+    end
+  end
+
+  properties (SetAccess = protected, GetAccess = public)
+    x0
+    hand_right = 0;
```
Comment (RussTedrake) [inline drake/examples/Valkyrie/Valkyrie.m]: zap me

**Comment by mauricefallon** [comment]
I've zapped the unused code as requested

File: drake/examples/Valkyrie/urdf/model/materials/textures/CMakeLists.txt [hunk 7]
```diff
@@ -0,0 +1,4 @@
+set (files
```
Comment (RussTedrake) [inline drake/examples/Valkyrie/urdf/model/materials/textures/CMakeLists.txt]: this seems problematic for mainline drake.

**Comment by RussTedrake** [comment]
thanks!  (sorry I missed the gazebo install on my first review).

**Comment by mauricefallon** [comment]
No worries. I've just removed the CMakeLists.txt file.

(I was trying to keep the model as close to NASA's version as possible)

**Comment by rdeits** [comment]
Only drake006 picked this up; the rest of the build servers missed it. @mauricefallon can you close it and re-open?

**Pull request merged**